crates/uv-macros/src/lib.rs

```diff
@@ -49,3 +49,56 @@ fn impl_combine(ast: &DeriveInput) -> TokenStream {
     };
     gen.into()
 }
+
+fn get_doc_comment(attr: &Attribute) -> Option<String> {
+    if attr.path().is_ident("doc") {
+        if let syn::Meta::NameValue(meta) = &attr.meta {
+            if let syn::Expr::Lit(expr) = &meta.value {
+                if let syn::Lit::Str(str) = &expr.lit {
+                    return Some(str.value().trim().to_string());
+                }
+            }
+        }
+    }
+    None
+}
+
+#[proc_macro_attribute]
+pub fn collect_constants(_attr: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
```suggestion
pub fn env_vars_metadata(_attr: TokenStream, input: TokenStream) -> TokenStream {
```

Maybe rename this to align with the naming of the other macros and be more explicit that this is for env var metadata?